src/SIL.LCModel/DomainImpl/OverridesLing_MoClasses.cs

```diff
@@ -3434,7 +3430,7 @@ public override ITsString ShortNameTSS
 			get
 			{
 				var tss = Form.VernacularDefaultWritingSystem;
-				if (tss != null || tss.Length > 0)
+				if (tss.Length > 0)
```

Comment:
I actually think the previous behavior was correct and that string can totally be null, you could just do `tss?.Lengh > 0` though

Reply:
Thanks. It's weird that it didn't flag this. That's probably why I removed it